src/BSD-3-Clause.xml

```diff
@@ -5,14 +5,15 @@
       <crossRefs>
          <crossRef>https://opensource.org/licenses/BSD-3-Clause</crossRef>
          <crossRef>https://www.eclipse.org/org/documents/edl-v10.php</crossRef>
+         <crossRef>https://github.com/sphinx-doc/alabaster/blob/0.x/LICENSE</crossRef>
       </crossRefs>
       <notes>Note that the Eclipse Distribution License - v 1.0 (EDL 1.0) is a match to BSD-3-Clause, even though it uses a different name.</notes>
     <text>
       <copyrightText>
         <p>Copyright (c) &lt;year&gt; &lt;owner&gt;. </p>
       </copyrightText>
-
-        <p>Redistribution and use in source and binary forms, with or without modification, <alt match="are|is" name="tobe">are</alt> permitted provided
+        <p>Redistribution and use in source and binary forms<alt match="()|( of the theme)" name="theme"></alt>,
```

Comment:
I _think_ this works, but should also have a `spacing="none"` attribute on the alt tag. That's to cause it to not expect to have whitespace between the matched text and the preceding element ("forms") and subsequent element (the comma).